### ISSUE TITLE
Add CORS proxy for scene assets in local development

### ISSUE THREAD

**Issue by speigg** [EtherealEngine/etherealengine#8608]
We previously had the default behavior of copying scene assets into the local file system for local development, however this was inconvenient for obvious reasons. Now, projects that reference assets which block CORS requests make local development workflows impossible. We should use a proxy server in local development to avoid CORS issues.

**Comment by hanzlamateen** [comment]
I wonder is using MinIO will fix this issue?

**Comment by barankyle** [comment]
I don't think so, because the engine still won't by default copy external files to the local machine when running normally in a local environment.
